tordnscrypt/src/main/res/values/strings.xml

```diff
@@ -271,233 +271,233 @@
     <string name="pref_dnscrypt_import_ipblacklist">Import IP blacklist</string>
     <string name="pref_dnscrypt_import_ipblacklist_summ">Import IP blacklist. You can select multiple files, duplicate lines will be removed.</string>
     <string name="pref_dnscrypt_erase_ipblacklist">Erase IP blacklist</string>
 
-    <string name="pref_dnscrypt_whitelist_categ">Pattern-based whitelisting (blacklists bypass).</string>
+    <string name="pref_dnscrypt_whitelist_categ">Pattern-based whitelisting (blacklist bypass).</string>
     <string name="pref_dnscrypt_whitelist">Whitelist</string>
-    <string name="pref_dnscrypt_whitelist_summ">Whitelist support the same patterns as blacklists. If a name matches a whitelist entry, the corresponding session will bypass names and IP filters.</string>
-    <string name="pref_dnscrypt_import_whitelist">Import whitelist</string>
+    <string name="pref_dnscrypt_whitelist_summ">Whitelisting support the same patterns as blacklists. If a name matches a whitelist entry, the corresponding session will bypass its names and IP filters.</string>
+    <string name="pref_dnscrypt_import_whitelist">Import Whitelist</string>
     <string name="pref_dnscrypt_import_whitelist_summ">Import a domain list. You can select multiple files, duplicate lines will be removed.</string>
-    <string name="pref_dnscrypt_erase_whitelist">Erase whitelist</string>
+    <string name="pref_dnscrypt_erase_whitelist">Erase Whitelist</string>
 
     <string name="pref_dnscrypt_sources_categ">Servers</string>
     <string name="pref_dnscrypt_sources">Sources</string>
     <string name="pref_dnscrypt_sources_summ">Remote lists of available servers.</string>
-    <string name="pref_dnscrypt_refresh_delay" translatable="false">refresh_delay</string>
-    <string name="pref_dnscrypt_refresh_delay_summ">Sources list expiring after refresh_delay hours.</string>
+    <string name="pref_dnscrypt_refresh_delay" translatable="false">Refresh-Delay</string>
+    <string name="pref_dnscrypt_refresh_delay_summ">Lists of sources expire after an amount of hours set in \"Refresh Delay\".</string>
 
     <string name="pref_dnscrypt_relays_categ">Relays</string>
     <string name="pref_dnscrypt_relays">Sources</string>
-    <string name="pref_dnscrypt_relays_delay" translatable="false">refresh_delay</string>
-    <string name="pref_dnscrypt_relays_delay_summ">Relays list expiring after refresh_delay hours.</string>
+    <string name="pref_dnscrypt_relays_delay" translatable="false">Refresh-Delay</string>
+    <string name="pref_dnscrypt_relays_delay_summ">Lists of relays expire after an amount of hours set in \"Refresh-Delay\".</string>
     <string name="pref_dnscrypt_relays_title">DNSCrypt Relays</string>
 
 
-    <string name="pref_dnscrypt_not_exist">Preference does not exist in dnscrypt-proxy.toml!</string>
+    <string name="pref_dnscrypt_not_exist">The preference does not exist in dnscrypt-proxy.toml!</string>
```

Comment:
Just wanted to make sure this isn't "preference for", or possibly "a preference".